README.md

```diff
@@ -2,6 +2,20 @@
 
 Creates `.ova` file for `cloudstack` from `vagrant` box.
 
+## Requirements
+
+* ansible
+* VirtualBox
+* vagrant
+* bundler
+* [ovftool](https://my.vmware.com/web/vmware/details?downloadGroup=OVFTOOL400&productId=353) (registration required)
```

Comment:
the link should be `https://www.vmware.com/support/developer/ovf/` if there is no reason to use the fixed version.

Reply:
fixed in 5dc8a03